source/grid/tria.cc

```diff
@@ -1409,37 +1409,56 @@ namespace internal
                                  const unsigned int                       level_objects,
                                  internal::Triangulation::NumberCache<2> &number_cache)
       {
-        // update lines and n_levels
-        compute_number_cache (triangulation,
-                              level_objects,
-                              static_cast<internal::Triangulation::NumberCache<1>&>
-                              (number_cache));
+        // update lines and n_levels. since we don't access any
```

Comment:
I meant the variables in `number_cache`, but you're right that it is confusing. I clarified it.